src/test/java/org/broadinstitute/consent/http/service/LibraryCardServiceTest.java

```diff
@@ -350,6 +350,25 @@ public void testFindLibraryCardById() {
     assertEquals(result.getId(), libraryCard.getId());
   }
 
+  @Test
+  public void testAddDaaToLibraryCard() {
```

Comment:
Similar to above tests, these don't have any assertions. For the most part, these are just pass-throughs to the underlying query so we don't need very much, perhaps just a simple assertion:
```
assertDoesNotThrow(() -> service.addDaaToLibraryCard(libraryCard.getId(), 1));
```